ui/src/org.ts

```diff
@@ -176,15 +176,54 @@ export const memberHandleValidationStore = (
   ]);
 };
 
+// Check whether a transaction Message involves a specific org.
+const involvesOrg = (
+  message: transaction.Message,
+  orgId: string,
+  accountId: string
+): boolean => {
+  switch (message.type) {
+    case transaction.MessageType.ProjectRegistration:
+      return (
+        message.domainType === project.Domain.Org && message.domainId == orgId
+      );
+    // TODO(nuno): Include Transfer when receipient is the given org, from which we need the address.
+    case transaction.MessageType.OrgRegistration:
+    case transaction.MessageType.OrgUnregistration:
+      return message.id == orgId;
+    case transaction.MessageType.Transfer:
+      return message.recipient == accountId;
+    case transaction.MessageType.TransferFromOrg:
+    case transaction.MessageType.MemberRegistration:
+    case transaction.MessageType.MemberUnregistration:
+      return message.orgId === orgId;
+    default:
+      return false;
+  }
+};
+
+// Obtain the transactions that involve an org with the given `id` and `accountId`.
 export const orgTransactions = (
   transactions: [transaction.Transaction],
-  id: string
+  id: string,
+  accountId: string
 ): [transaction.Transaction] | transaction.Transaction[] => {
-  return transactions.filter(tx => {
-    return tx.messages[0].type ===
-      transaction.MessageType.ProjectRegistration &&
-      tx.messages[0].domainType === project.Domain.Org
-      ? tx.messages[0].domainId == id
-      : null;
-  });
+  return transactions.filter(tx => involvesOrg(tx.messages[0], id, accountId));
+};
+
+export const fallback = (): Org => {
```

Comment:
What is a fallback org?

Reply:
We should document this. When we look at transfer records involving an org that is no longer around (on the local machine), we need data to display the avatar component of such entities. To reproduce, register an org, transfer something from it to the (registered) user, restart the app, and now look at the `Incoming Transfer` in the user wallet. The org is gone (flushed on restart). It will show it as:

![Screenshot from 2020-08-03 10-25-37](https://user-images.githubusercontent.com/7075260/89161995-af772d80-d573-11ea-824f-218d743dda14.png)